vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopWallet.kt

```diff
@@ -231,9 +235,10 @@ class OidcSiopWallet(
         )
         val jwsPayload = idToken.serialize().encodeToByteArray()
         val jwsHeader = JwsHeader(algorithm = JwsAlgorithm.ES256)
```

Comment:
should we not take this opportunity to get rid of all the hardcoded stuff and at least make it a constructor property, that defaults to ES256? I'm not saying this PR should do it, but before we release 3.4.0, those hardcoded values should go.

Reply:
But in `JwsHeader`, the `algorithm` parameter is required, i.e. non-null.